internal/core/command/errors/types.go

```diff
@@ -26,3 +26,28 @@ func (e ErrCommandNotAssociatedWithDevice) Error() string {
 func NewErrCommandNotAssociatedWithDevice(commandID string, deviceID string) error {
 	return ErrCommandNotAssociatedWithDevice{commandID, deviceID}
 }
+
+type ErrExtractingInfoFromRequest struct {
+	commandID string
+	deviceID  string
```

Comment:
Are these useful? If you cannot extract the information from the request then I do not see how we can provide this information. This goes with my existing comment regarding this `error` always being constructed with empty strings